The Globe and Mail.js

```diff
@@ -1,223 +1,221 @@
-		if (metaTags["article_keywords"]) {
-			tagsContent = metaTags["article_keywords"].split("; ");
-		}
-		
-		for (var i = 0; i < tagsContent.length; i++) {
-		     	if (tagsContent[i] != (" ") && tagsContent[i] != ("")) {
-			     	newItem.tags[i] = tagsContent[i];
-		     	}
-	     	}	
-		
-		associateMeta (newItem, metaTags, "headline", "title");
-		associateMeta (newItem, metaTags, "summary", "abstractNote");
-		associateMeta (newItem, metaTags, "desk", "section");	
-		associateMeta (newItem, metaTags, "article_id", "callNumber");
-		associateMeta (newItem, metaTags, "credit", "rights");
+	newItem = new Zotero.Item("newspaperArticle");
+	newItem.url = url;
+
+	//get headline
+	var title = ZU.xpathText(doc, '//article/h1');
+	if (!title) title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	newItem.title = title;
+
+	//get abstract
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@property="og:description"]/@content');
 	
-	//rest of author (shared between both newspaperArticle types)
-		if (author) {
-			
-		
-			if (author.substr(0,3).toLowerCase() == "by ") {
-				author= author.substr(3);
-			}
-		
-			var authors = author.toLowerCase().split(" and ");
-			for (var i=0; i<authors.length; i++) {
-				var author = authors[i];
-				var words = author.split(" ");
-				
-				for (var j=0; j<words.length; j++) {
-					if (words[j] != "") {
-						words[j] = words[j][0].toUpperCase() + words[j].substr(1).toLowerCase();
-					}
-				}
-				author = words.join(" ");
-				newItem.creators.push(Zotero.Utilities.cleanAuthor(author, "author"));			
-			}
-		}
-	}	
+	//get date
+	var xpathdate = '//time[@itemprop="datePublished"]';
+	var date = ZU.xpathText(doc, xpathdate);
+	if (date) {
+		newItem.date = ZU.strToISO(date);
+	}
+
+	//get author
+	var authors = ZU.xpath(doc, '//meta[@itemprop="author"]/@content');
+	for (var i = 0; i<authors.length; i++){
+		newItem.creators.push(ZU.cleanAuthor(authors[i].textContent, "author"));
+	}
+
+	var publishers = ZU.xpath(doc, '//article/header//div[@itemprop="publisher"]//p');
+	for (var i = 0; i<publishers.length; i++){
+		newItem.creators.push( {
+		"lastName" : publishers[i].innerText,
+		"creatorType" : "contributor",
+		"fieldmode" : 1
+		});
+	}
+
+	newItem.language = ZU.xpathText(doc, '//meta[@http-equiv="Content-Language"]/@content');
 	
-	newItem.url = doc.location.href;
+	newItem.section = ZU.xpathText(doc, '//meta[@name="article:type"]/@content');
+
+	var xpathtags = ZU.xpathText(doc, '//meta[@name="keywords"]/@content');
+	newItem.tags = xpathtags.split(";").filter(function(tag) {return tag.length != 0});
+
+	var related_links = ZU.xpathText(doc, '//article//li/p/a/@href');
+	var note=""
+	if(related_links)
+		note = addNote(note, related_links.split(",").filter(function(tag) {return tag.length != 0}))
+	if (note.length != 0) {
+		newItem.notes.push( {note: note} );
+	}
+
+	newItem.attachments = ({
+		document: doc,
+		title: "The Globe and Mail Snapshot",
+		mimeType: "text/html"
```

Comment:
try if this works better w/o mimeType -- I don't think it should matter, but you also don't need a mimeType when saving the current doc.